content/en/community.md

```diff
@@ -44,6 +44,12 @@ On this list please use bottom posting, reply to the list (rather than to
 another sender), and don't reply to digests. A searchable archive of this list
 is available [here](https://mail.python.org/archives/list/scipy-dev@python.org/).
 
+### [SciPy Slack space]
```

Comment:
To follow the other items, this can be a link pointing to the invite. It's ok to have it repeated I would say.
```suggestion
### [SciPy Slack space](https://join.slack.com/t/scipy-community/shared_invite/zt-1a76bomjr-fuS1ZTnmP7b32kIhLb6QMg)
```